src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,30 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            className={classNames({
+              'is-active': selectedTabId === tab.id,
+            })}
+            data-cy="Tab"
+            key={tab.id}
+          >
+            <a
+              href={`#${tab.id}`}
+              data-cy="TabLink"
+              onClick={() => onTabSelected(tab.id)}
```

Comment:
Don't call the callback if the tab was not changed - you should check if a selected tab is not the same one you've clicked